packages/tracing/test/index.test.ts

```diff
@@ -0,0 +1,23 @@
+import { getCurrentHub } from '@sentry/hub';
+
+import { BrowserTracing, Integrations } from '../src';
+
+describe('index', () => {
+  describe('Integrations', () => {
+    it('is exported correctly', () => {
+      Object.keys(Integrations).forEach(key => {
+        expect(Integrations[key as keyof typeof Integrations].id).toStrictEqual(expect.any(String));
+      });
+    });
+
+    it('contains BrowserTracing', () => {
+      expect(Integrations.BrowserTracing).toEqual(BrowserTracing);
+    });
+  });
+
+  it('patches the global hub as a side effect', () => {
```

Comment:
```suggestion
  test('importing from `index.ts` adds an implementation for `Hub.startTransaction` as a side effect', () => {
```

Also, how would you feel about putting this test before the inner `describe`? It took me a second to realize it was only in the outer one, and I think moving it would make it clearer.

Reply:
Good suggestion, will do.